src/zope/app/container/browser/tests/test_contents.py

```diff
@@ -374,5 +378,5 @@ def test_suite():
         makeSuite(TestCutCopyPaste),
         ))
 
-if __name__=='__main__':
+if __name__=='__main__': # pragma: no cover
     main(defaultTest='test_suite')
```

Comment:
Some of them do. That one does, for example, but `test_contents_functional` does not.